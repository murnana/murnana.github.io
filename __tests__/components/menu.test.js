import React from "react"
import renderer from "react-test-renderer"
import Menu from "../../src/components/menu"

describe("Menu", () => {
  it("render", () => {
    const tree = renderer.create(<Menu />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
