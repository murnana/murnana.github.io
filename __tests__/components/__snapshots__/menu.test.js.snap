// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu render 1`] = `
<button
  className="menu"
  onClick={[Function]}
>
  Menu
</button>
`;
